app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -93,11 +96,6 @@ export default function CompetingStep({
           payment: registration.payment,
         },
       );
-      // Going from pending -> Cancelled
-      if (data.registration.competing.registration_status === 'cancelled') {
-        dispatch(setMessage('competitions.registration_v2.register.registration_status.cancelled', 'positive'));
-        return nextStep({ toStart: true });
-      }
```

Comment:
Why is this transition/step being dropped?

Reply:
because deletion is now handled on the registration overview